timezones.json

```diff
@@ -38,8 +38,8 @@
   },
   {
     "value": "Alaskan Standard Time",
-    "abbr": "AKDT",
-    "offset": -8,
+    "abbr": "AKST",
```

Comment:
@Pablodotnet thanks for PR.

- I see that for `Alaskan Standard Time` it accept two abbr `AKDT` and `AKST` for summer time and for a winter time